asf_welsh_energy_consultation/getters/get_data.py

```diff
@@ -139,14 +229,55 @@ def get_rurality():
     return oa_rural
 
 
+def check_local_epc():
+    """
+    Checks local directory for relevant EPC batch and downloads relevant EPC batch to local directory if not found.
+
+    """
+    epc_batch = arguments.epc_batch
+
+    local_epc_batch_path = os.path.join(
+        LOCAL_DATA_DIR,
+        base_config.OUTPUT_DATA_PATH,
+        epc_batch.upper().replace("COMPLETE", "complete"),
+    )
+
+    local_epc_path = os.path.join(
+        local_epc_batch_path, f"EPC_GB_{epc_processing_version}.csv"
+    )
```

Comment:
When I print this path it gives me:
`/Users/anasofiapinto/Documents/ASF data/outputs/EPC/preprocessed_data/NEWEST/EPC_GB_preprocessed.csv`

hence, it's looking for the file in an incorrect path, that's why it can't find it. It's trying to find it in 'NEWEST' instead of the correct quarter folder.

It should be trying to find this path:
`/Users/anasofiapinto/Documents/ASF data/outputs/EPC/preprocessed_data/2023_Q2_complete/EPC_GB_preprocessed.csv`

Reply:
not sure if the same happens when you specify the batch - haven't tested it

Reply:
have updated this and tested locally. should work now but please do run again to test :)

Reply:
it worked now :)